manifests/params_manager.pp

```diff
@@ -5,7 +5,7 @@
     'Linux': {
 
     # Installation
-      $server_package_version                          = '4.2.1-1'
+      $server_package_version                          = '4.2.2'
```

Comment:
Same here.

Reply:
This change was made in commit https://github.com/wazuh/wazuh-puppet/commit/bfab4c0c1bf761306dc9882808ddcfb44d3d846d

Reply:
Fixed in commit https://github.com/wazuh/wazuh-puppet/pull/393/commits/7342c3713d0d64202e9a9dc49c53f4bf4887505f